### PR TITLE
Easier to read CSS style

### PR DESCRIPTION
**Pull request by es20490446e**
Screenshot 1
Screenshot 2

### CODE REVIEW

File: roffit [hunk 1]
```diff
@@ -584,7 +602,7 @@ parsefile($filename);
 
 my @conv = linkfile();
 
-my $title=sprintf("%s man page",
+my $title=sprintf("%s manual",
                   $manpage{'name'}?$manpage{'name'}:"secret");
```
Comment (bagder) [inline roffit:606]: This is not a CSS change

File: roffit [hunk 2]
```diff
@@ -598,18 +616,9 @@ MOO
     ;
     defaultcss();
     print "</head><body>\n";
-}
-else {
-    print "<!-- generated with roffit -->\n";
+    print "</body></html>\n";
```
Comment (bagder) [inline roffit:619]: This is not a CSS change

**Comment by es20490446e** [comment]
https://youtu.be/O6bShHdzWwo

**Comment by es20490446e** [comment]
So are you interested in this, or shall I create my own derivative version?

**Comment by bagder** [comment]
Sorry, I didn't see any motivation. Please provide feedback as text, then everyone can read it, online, offline over email etc.

Why does the default embedded CSS style matter at all? Won't you use your completely own stylesheet anyway and customize it exactly to your own preferences?

**Comment by es20490446e** [comment]
I don't see how to use a custom CSS style, except by modifying it in the roffit code itself. Excuse my inexperience on web development.

**Comment by bagder** [comment]
When you want to integrate roffit-generated contents into your site I recommend you use `--bare` and provide your own stylesheet that makes the content look exactly the way you want.

We can't expect everyone to like or even approve of the default look, however we make it look.

**Comment by es20490446e** [comment]
Anyway I would consider this style as default. It's your choice.

If you decide otherwise let me know soon, so I can put the changes in my own software.

**Comment by bagder** [comment]
Changing roffit itself just to get a different style is the wrong approach. It is designed to let you define your own. Why is that not good enough?

**Comment by es20490446e** [comment]
I have the sensation that you aren't really considering what I'm telling you 😏

**Comment by bagder** [comment]
You want to change the default styles. I ask you to further motivate why that's necessary.

**Comment by es20490446e** [comment]
That the proposed default is a better default. And will be good enough for many people not needing to provide their own CSS style, what doesn't happen with the current one.

**Comment by bagder** [comment]
I disagree with that characterization of the existing default style. It might not be good-looking but its also a matter of personal preference and I don't think your proposed change makes it objectively better. Just different. And for different, we already allow every user to set their own style.

Are you using any metric or scientific way to judge "easier to read" ?

**Comment by es20490446e** [comment]
You see, it's not a matter of how much explanation I provide.

Either you like it or you don't, life is that easy.

**Comment by bagder** [comment]
Sorry, I don't like it. Then I presume this case is closed.

**Comment by es20490446e** [comment]
Yeap, like that. We can skip the infinite talking.

**Comment by es20490446e** [comment]
Sorry, but the ergonomics of roffit right now aren't good enough.

It needs to have a good default style, and specially not saying "generated with roffit" at the bottom.

So either these get fixed in roffit, or they will become their own fork.

**Comment by bagder** [comment]
How do you judge what's a "good default style" ?

**Comment by es20490446e** [comment]
- Easy to differentiate parts.
- Comfortable for the eyes, no big differences in luminance between dark and bright areas.

**Comment by es20490446e** [comment]
Let me know what is your choice in a couple of days, otherwise I would make a fork.

**Pull request closed without merging**